compiler/catala_utils/pos.mli

```diff
@@ -59,5 +59,13 @@ val format_loc_text : Format.formatter -> t -> unit
 (** Open the file corresponding to the position and retrieves the text concerned
     by the position *)
 
+val format_loc_text_parts :
+  t ->
+  (Format.formatter -> unit)
+  * (Format.formatter -> unit)
+  * (Format.formatter -> unit) option
+(** Like [format_loc_text], but returns the printing functions in 3 separate
```

Comment:
If it's not used anymore, can you delete the old `format_loc_text`?

Reply:
It's still used for now, when displaying traces for example